protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/StepContainer.tsx

```diff
@@ -249,7 +253,16 @@ export function StepContainer(props: StepContainerProps): JSX.Element {
             ) : null}
           </Flex>
         </Btn>
-      </Box>
+        {dragHovered ? (
+          <Flex
+            data-testid={`${stepId}-divider`}
+            backgroundColor={COLORS.blue50}
+            height="0.25rem"
+            width="100%"
+            borderRadius={BORDERS.borderRadius2}
+          />
```

Comment:
```suggestion
<Divider
            marginY="0"
            data-testid={`${stepId}-divider`}
            backgroundColor={COLORS.blue50}
            height="0.25rem"
            width="100%"
            borderRadius={BORDERS.borderRadius2}
          />
```

I guess the above looks same as `Flex`. If you see any diff, please ignore this.

![Screenshot 2024-10-31 at 7 41 27 PM](https://github.com/user-attachments/assets/fa3da2de-35e3-4613-9280-e65dcf8395de)